Added --ignore-keyboard

gebbber opened the pull request:
When using `-w` in `.bashrc` file as suggested by help text for `--single-wave`, keyboard input is captured, preventing the user from starting to type a terminal command. This produces the visual effect to run on opening the terminal, but quickly becomes impractical for everyday use as the user needs to wait for it to complete before starting to type a command.

This new command line option `-i` or `--ignore-keyboard` allows the user to specify that keyboard input is ignored. Any keypresses while the script is running will be available to the terminal on exit.

Previous pull request required that `--single-save` or `--time` options be used concurrently, but this was unnecessary, as the main loop still allows `SIGINT` capture.

gebbber committed:
Added --ignore-keyboard

will8211 commented:
Hi @gebbber.

Thanks for this improvement! I'll have a look this week and test it out cross-platform.

gebbber committed:
Update unimatrix.py

Removed documentation artifact from previous version (`--ignore-keyboard` was previously only allowed with `-w` or `-t`)

gebbber commented:
Sounds good. I left an artifact in the `--help` docs from my previous version. I've just updated, as I had already removed the requirement to use it with `-t` or `-w`. (might be worth checking that Ctrl-C works cross-platform)

will8211 reviewed (approved):
Looks good
